src/routes/invidious_routes/download.ts

```diff
@@ -0,0 +1,87 @@
+import type { Context, Hono } from "hono";
+import { HTTPException } from "hono/http-exception";
+import { verifyRequest } from "../../lib/helpers/verifyRequest.ts";
+
+export default function getDownloadHandler(app: Hono) {
+    async function handler(c: Context) {
+        const body = await c.req.formData();
+
+        const videoId = body.get("id") as string | undefined;
```

Comment:
```suggestion
        const videoId = body.get("id")?.toString();
```
This avoids overriding the type manually and results in the same outcome.